DistributionSystemApi/DistributionSystemApi/MailTest.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Net.Mail;
+using System.Net;
+using MailLibrary;
+using DistributionSystemApi.MailLibrary;
+
+namespace DistributionSystemApi
+{
+    public class MailTest
+    {
+        public async void TestSend()
```

Comment:
please delete this file if it was created for test purpose, or better create new test project and write several test cases for this service(if you have any questions here, contact me and we will discuss)